BlueDB/src/main/java/org/bluedb/api/BlueDbVersion.java

```diff
@@ -0,0 +1,9 @@
+package org.bluedb.api;
+
+public enum BlueDbVersion {
+	v_1_0_0,
+	v_1_0_1,
+	v_1_1_0;
+	
+	public static BlueDbVersion CURRENT = v_1_1_0;

```

Comment:
Could you pull in master and update the versions. There should be 1.0.0, 1.0.1, 1.0.2, and now 2.0.0. I don't know for sure if we'll want to forever list all possible versions in here, but for now its fine. Eventually we might just want the current version and a way to parse and compare versions. That way we can check if a version is before or after a specific version without having to have an enum with every possibility.

Reply:
Do you want to limit to major and minor?  If we make some potentially breaking change where we'll actually need to know the existing version, it should at least change minor versions.

Reply:
Alternative, we could do major versions and add minor versions only if really necessary.

Reply:
Actually, we've already tagged 2.0.0, so when this gets officially released it'll probably be 2.0.1. Sometime soon we'll need to create a checklist of tasks that need to be done when we're ready to tag a new version.